2020-02-Spring/3_trees/README.md

```diff
@@ -23,21 +23,111 @@ Problems and solutions for Trees session on May 8, 2020.
 
 <a name="p1"/>
 
-### 1. PROBLEM 1 TODO :bug:
+### 1. Lowest Common Ancestor
```

Comment:
This is a very well written solution, IMO! :fire: :tada: 

Just had a few tiny nitpicky suggestions! Feel free to ignore as they are not critically important.

Reply:
Thank you! Your suggestions are good stuff, will make some edits along those lines